ui/page/root/wallet_list.go

```diff
@@ -285,58 +285,57 @@ func (pg *WalletSelectorPage) walletWrapper(gtx C, item *load.WalletItem) D {
 						}),
 					)
 				}),
-				layout.Rigid(func(gtx C) D {
-					return layout.Flex{
-						Axis:      layout.Horizontal,
-						Alignment: layout.Middle,
-					}.Layout(gtx,
-						layout.Rigid(func(gtx C) D {
-							return pg.syncStatusIcon(gtx, item.Wallet)
-						}),
-						layout.Rigid(func(gtx C) D {
-							if len(item.Wallet.GetEncryptedSeed()) > 0 {
-								return layout.Flex{
-									Axis:      layout.Horizontal,
-									Alignment: layout.Middle,
-								}.Layout(gtx,
-									layout.Rigid(func(gtx C) D {
-										return layout.Inset{
-											Left:  values.MarginPadding8,
-											Right: values.MarginPadding8,
-										}.Layout(gtx, pg.Theme.Icons.Dot.Layout8dp)
-									}),
-									layout.Rigid(func(gtx C) D {
-										return layout.Inset{
-											Right: values.MarginPadding4,
-										}.Layout(gtx, pg.Theme.Icons.RedAlert.Layout16dp)
-									}),
-									layout.Rigid(pg.Theme.Label(values.TextSize16, values.String(values.StrNotBackedUp)).Layout),
-								)
-							}
-							return D{}
-						}),
-					)
+				layout.Flexed(1, func(gtx C) D {
+					return layout.E.Layout(gtx, func(gtx C) D {
+						return layout.Flex{
+							Axis:      layout.Vertical,
+							Alignment: layout.End,
+						}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								return components.LayoutBalanceWithStateSemiBold(gtx, pg.Load, item.TotalBalance.String())
+							}),
+							layout.Rigid(func(gtx C) D {
+								usdBalance := ""
+								if components.IsFetchExchangeRateAPIAllowed(pg.WL) {
+									usdBalance = utils.FormatAsUSDString(pg.Printer, item.TotalBalance.MulF64(pg.assetRate[item.Wallet.GetAssetType()]).ToCoin())
+								}
+								return components.LayoutBalanceWithStateUSD(gtx, pg.Load, usdBalance)
+							}),
```

Comment:
Is it possible to use methods graciously to give meaning to the layout code used in this PR? For e.g., the highlighted code could be:

```
components.FlexLayout(axis, alignment, []widgets) {
	flexChildren := make([]layout.Rigid, len(widgets))
	for i, widget := range widgets {
		flexChildren[i] = layout.Rigid(widget)
	}

	return layout.Flex{
		Axis:      axis,
		Alignment: alignment,
	}.Layout(gtx, flexChildren...)
}
```

Reply:
`layout.flex` is already an upstream component, wrapping it into our component seems unnecessary. 

you don't think the current upstream methods have much meaning?

Reply:
this is in components.go
```
func FlexLayout(gtx C, options FlexOptions, widgets []func(gtx C) D) D {
	flexChildren := make([]layout.FlexChild, len(widgets))
	for i, widget := range widgets {
		flexChildren[i] = layout.Rigid(widget)
	}

	return layout.Flex{
		Axis:      options.Axis,
		Alignment: options.Alignment,
	}.Layout(gtx, flexChildren...)
}
```

this is in wallet list file
```
					return layout.E.Layout(gtx, func(gtx C) D {
						widgets := []func(gtx C) D{
							func(gtx C) D {
								return components.LayoutBalanceWithStateSemiBold(gtx, pg.Load, item.TotalBalance.String())
							},
							func(gtx C) D {
								usdBalance := ""
								if components.IsFetchExchangeRateAPIAllowed(pg.WL) {
									usdBalance = utils.FormatAsUSDString(pg.Printer, item.TotalBalance.MulF64(pg.assetRate[item.Wallet.GetAssetType()]).ToCoin())
								}
								return components.LayoutBalanceWithStateUSD(gtx, pg.Load, usdBalance)
							},
						}
						options := components.FlexOptions {
							Axis: layout.Vertical,
							Alignment: layout.End,
						}
						components.FlexLayout(gtx, options, widgets)
```

this is what you're referring to right?

Reply:
Yeah. Or perhaps:
```
					return layout.E.Layout(gtx, func(gtx C) D {
						options := components.FlexOptions {
							Axis: layout.Vertical,
							Alignment: layout.End,
						}
						components.FlexLayout(gtx,
							options,
							func(gtx C) D {
								return components.LayoutBalanceWithStateSemiBold(gtx, pg.Load, item.TotalBalance.String())
							},
							func(gtx C) D {
								usdBalance := ""
								if components.IsFetchExchangeRateAPIAllowed(pg.WL) {
									usdBalance = utils.FormatAsUSDString(pg.Printer, item.TotalBalance.MulF64(pg.assetRate[item.Wallet.GetAssetType()]).ToCoin())
								}
								return components.LayoutBalanceWithStateUSD(gtx, pg.Load, usdBalance)
							},
						)
```

with the components fn using a spread
```
func FlexLayout(gtx C, options FlexOptions, widgets ...func(gtx C) D) D {
```

Reply:
But I see it's basically the same thing, so maybe this can't be refactored.

Reply:
Yeah, I'd push the change above then